draft-ietf-rtcweb-jsep.xml

```diff
@@ -6729,7 +6729,12 @@ a=msid:bbce3ba6-abfc-ac63-d00a-e15b286f8fce
           each SSRC indicated in the packet that is found in the
           incoming SSRC table, first deliver a copy of the packet to
           the "m=" line associated with that SSRC, but then remove the
-          entry for that SSRC from the incoming SSRC table.</t>
+          entry for that SSRC from the incoming SSRC table after an
+          appropriate delay (Similar to <xref target="RFC3550" />
```

Comment:
Should that be "as specified in 3550"? We're not adding new behavior here, we're reminding people what 3550 says on how to handle BYE.